content/pt-br/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Utilize o Cilium para NetworkPolicy
+content_type: task
+weight: 30
+---
+
+<!-- overview -->
+Essa página mostra como utilizar o Cilium para NetworkPolicy.
+
+Para saber mais sobre o Cilium, leia o artigo [Introdução ao Cilium (em inglês)](https://docs.cilium.io/en/stable/overview/intro).
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+## Fazendo o Deploy do Cilium no Minikube para Testes Básicos
+
+Para familiarizar-se com o Cilium você poderá seguir o guia [Guia de Primeiros Passos do Cilium no Kubernetes (em inglês)](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/) e realizar uma instalação básica do Cilium através de um DaemonSet no minikube.
+
+Inicie o minikube, a versão mínima exigida é >= v1.5.2, com os seguintes argumentos:
+
+```shell
+minikube version
+```
+```console
+minikube version: v1.5.2
+```
```

Comment:
Minor: this block's formatting isn't matching the original page formatting
````suggestion
```
minikube version: v1.5.2
```
````